tests/system/conftest.py

```diff
@@ -38,18 +66,9 @@ def use_mtls():
 
 
 @pytest.fixture(scope="session")
-def credentials(use_mtls):
```

Comment:
Does this change make the `use_mtls` fixture obsolete?  Looks like the `bq_client` and `local_shakespeare_table_reference` fixtures still use it, at least to skip.

Reply:
We'll still need to use it to skip, yeah. But since I don't have local service account key files anymore on my work laptop, I wanted to make sure we could always run the tests with default credentials.